packages/fraction-model/configure/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@pie-element/fraction-model-configure",
+  "private": true,
+  "version": "2.18.1",
```

Comment:
we should have an initial version like 1.0.0

Reply:
Updated